cling/dict/ROOT-7879/CMakeLists.txt

```diff
@@ -1,12 +1,19 @@
 # ROOT-7879
 ROOTTEST_ADD_TEST(skipLinkDefDict
-                  COMMAND ${ROOT_rootcling_CMD} -f libSkippingDict.cxx -rmf libSkipping.rootmap -rml libSkipping.so  ${CMAKE_CURRENT_SOURCE_DIR}/skippingHeader.h ${CMAKE_CURRENT_SOURCE_DIR}/skipLinkDef.h
+                  COMMAND ${ROOT_rootcling_CMD} -f libSkippingDict.cxx -rmf libSkipping.rootmap -rml libSkipping${libsuffix} ${CMAKE_CURRENT_SOURCE_DIR}/skippingHeader.h ${CMAKE_CURRENT_SOURCE_DIR}/skipLinkDef.h
                   OUTREF skipLinkDefDict.ref)
 
+if(MSVC)
+    set(grep_cmd "findstr")
```

Comment:
This is used twice, could it be factor out in the cmake/* files.?

Reply:
Yes, but we also have grep with different arguments... Anyway, I'll move it elsewhere

Reply:
Done